openlibrary/core/vendors.py

```diff
@@ -60,14 +59,14 @@ def _get_amazon_metadata(isbn):
         'languages': list(product.languages),  # needs to be normalized
         'publishers': [product.publisher],
         'cover': product.large_image_url,
-        'isbn_10': [isbn_10],
-        'isbn_13': [isbn_13]
+        'isbn_10': [isbn if len(isbn) == 10 else isbn_13_to_isbn_10(isbn)],
```

Comment:
this could break if the isbn13 has no isbn10 equivalent

Reply:
@mekarpeles how is that possible? Don't all isbn10 values map to a bookland isbn13?